@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1574,6 +1574,7 @@ export class TextElementsRenderer {
     ): boolean {
         const pointLabel: TextElement = labelState.element;
         const textRenderState: RenderState | undefined = labelState.textRenderState;
+        const isRoadShield = iconIndex !== undefined;
```

Comment:
Just a minor thing: This name brings more meaning to this as the condition gives, shouldn´t it be just something like "hasIcon"

Reply:
This constant is to differentiate between a point label or a line marker. Better name it "isLineMarker" to use the same naming as in the rest of the code.

Reply:
renamed to "isLineMarker"